charts/intel-qat-resource-driver/README.md

```diff
@@ -46,3 +48,21 @@ You may also run `helm show values` on this chart's dependencies for additional
 | image.tag | string | `"v0.1.0"` |
 
 If you change the image tag to be used in Helm chart deployment, ensure that the version of the container image is consistent with deployment YAMLs - they might change between releases.
+
+
+## Troubleshooting read-only file system error for QAT
```

Comment:
Troubleshoot = "discovering why something does not work effectively and making suggestions about how to improve it"

But this section does not really tell how to root-cause the problem.

```suggestion
## Read-only file system error for QAT
```